src/js/solo/top-listens/TopListeningData.js

```diff
@@ -74,6 +100,40 @@ class TopListeningData extends Component {
     });
   }
 
+  /**
+   * Auto-fills a playlist with a few tracks from the user's top listening data
+   * This will use the addTrackHandler passed in to add the tracks to
+   * the user's goldify playlist
+   */
+  autoFillGoldifyPlaylist() {
+    alert(
```

Comment:
I've added in quite a few alerts, one in this PR and two in previous PRs - I'd like to instead replace these with user input, such as Yes/No buttons, but may want some guidance or discussion on how to do that.

Reply:
Definitely. I feel like this could be a quick peer-programming session to get this in. I think we touched on this a bit on the drive to the airport if I recall